### DISCUSSION TITLE
Merge GitHub Organisations

### DISCUSSION THREAD

**Discussion by Ornitor1210** [community/community#124983] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

We recently acquired a company, and we need to merge their repositories, and issues, and pull requests with ours. We would prefer if it can still be separate from our existing Repositories, and issues/pull request etc. since the code are for two different systems and applications. Can someone help with articles, or if this is even possible?

**Comment by gitstua** [comment, marked as answer]
👋🏽  HI @Ornitor1210 you can transfer repos - the [docs](https://docs.github.com/en/enterprise-cloud@latest/repositories/creating-and-managing-repositories/transferring-a-repository) explain this

Be sure to read the pre-requisites and [what's transferred ](https://docs.github.com/en/enterprise-cloud@latest/repositories/creating-and-managing-repositories/transferring-a-repository#whats-transferred-with-a-repository)

Maybe create a test repo in the source organization and test transferring just this to see if it works

An alternative if you have an Enterprise is to [transfer the existing organization](https://docs.github.com/en/enterprise-cloud@latest/organizations/managing-organization-settings/transferring-organization-ownership) under your enterprise. Again I would suggest a test of this to see if it moves everything you need.
